safety_limiter_msgs/msg/SafetyLimiterStatus.msg

```diff
@@ -0,0 +1,11 @@
+int8 NORMAL = 0
+int8 COLLISION_IS_PREDICTED = 1
+int8 WATCHDOG_TIMED_OUT = 3
+
+Header header
+int8 status
+
+float64 limit_ratio
+bool is_cloud_available
+bool has_watchdog_timed_out
+time stuck_started_since
```

Comment:
meaning of zero value (not being stuck) should be described as a comment.

Reply:
added